docs/networking.md

```diff
@@ -0,0 +1,67 @@
+# Advanced networking configurations
+
+It is possible to deploy this sidecar module to different networking configurations to attend different needs.
+
+For testing and evaluation purposes, it is very common that customers will deploy a public sidecar
+(public load balancer and public ECS tasks), but this is not a recommended approach for a production
+environment. In production, typically customers will deploy an entirely private sidecar (private load 
+balancer and private ECS tasks), but sometimes it is necessary to deploy a public load balancer and
+keep ECS tasks in a private subnet. It is also common that more than 5 ports are exposed in a
+production sidecar requiring special attention to the number of services and tasks that will
+be created for that.
+
+See the following sections how to set up the necessary parameters for each of these scenarios.
+
+All resources outlined below are expected to live in the same VPC, meaning that the parameter
+`vpc_id` will correspond to the ID of the VPC of all subnets used throughout the deployment 
+configuration.
+
+## Opening more than 5 ports
+
+ECS imposes a hard limit of 5 target groups per service. This means that if you need to
```

Comment:
should probably make some mention that this doesnt work for over 50 due to the NLB limits

Reply:
Fixed in [095eb0e](https://github.com/cyral-quickstart/quickstart-sidecar-aws-ecs/pull/13/commits/095eb0e289bcfa3c8e982dc6bace23e682613f4c).